Riot/Modules/Room/RoomViewController.h

```diff
@@ -156,6 +164,16 @@ extern NSNotificationName const RoomGroupCallTileTappedNotification;
  */
 + (instancetype)instantiateWithConfiguration:(RoomDisplayConfiguration *)configuration;
 
+/**
+ Creates and returns a new `RoomViewController` object.
+ 
+ @param configuration display configuration for the room view controller.
+ @param session The Matrix session.
+ 
+ @return An initialized `RoomViewController` object.
+ */
++ (instancetype)instantiateWithConfiguration:(RoomDisplayConfiguration *)configuration session:(nonnull MXSession*)session;
```

Comment:
I think this added method is not used- can you please check and remove it?

Reply:
it used in [roomCoordinator](https://github.com/vector-im/element-ios/blob/5eba40290e1504945d8e00b8b728a9186ac7764d/Riot/Modules/Room/RoomCoordinator.swift#L85)